launcher/ui/themes/ThemeManager.cpp

```diff
@@ -140,11 +140,6 @@ void ThemeManager::setApplicationTheme(const QString& name)
         auto& theme = themeIter->second;
         themeDebugLog() << "applying theme" << theme->name();
 
-        // applying the system theme initially seems to cause issues, so lets avoid that

```

Comment:
just noting down why I removed this, I had added it to try and fix #510 , which was fixed by #600 
So this actually just broke stuff while not fixing anything